### DISCUSSION TITLE
favicon.ico and favicon.svg become favicon.ico/ and favicon.svg/ when trailingslash = true which results in 404

### DISCUSSION THREAD

**Discussion by gangajamuna** [arthelokyo/astrowind#86] [Q&A] [answered]
Hi,
I'm not sure if I doing something wrong here.... favicon.ico and favicon.svg strings seem to be injected in the Header of the base Layout. But when trailingslash = true in the config file they seem to be injected with a slash at the end which results in 404 not found error. I'm not sure how I can change this... Can you give me a hint?
Tank you!

**Comment by gangajamuna** [comment, marked as answer]
Ahh ok, it came from the ExtraMetaTags.astro file... The function getRelativeLink was involved. I just wrote the correct links without the function and it works for me now...
Sorry

**Comment by prototypa** [comment]
Hi @gangajamuna 

That was a bug that **AstroWind** had, and had for a while. I had not noticed.
It also existed in the link to `/rss.xml`.

I fixed it:
- https://github.com/onwidget/astrowind/commit/e85d4061976db3482392cf6b427f50f05ab2b3e6
- https://github.com/onwidget/astrowind/commit/e276a990775f984fb579938d1eb2a8dca6a96624
- https://github.com/onwidget/astrowind/commit/6ea771991b7b68b6a1f46211c4f23588f0a64924

**Reply by gangajamuna** [reply]
Perfect. Thank you :)
